lochness/mediaflux/__init__.py

```diff
@@ -165,9 +173,9 @@ def sync_module(Lochness: 'lochness.config',
                             p = Popen(cmd, shell=True)
                             p.wait()
 
-                            # verify checksum after download completes
-                            # if checksum does not match, data will be downloaded again
-                            # ENH should we verify checksum 5 times?
+                            # verify checksum after download completes if
+                            # checksum does not match, data will be downloaded
+                            # again ENH should we verify checksum 5 times?
```

Comment:
Do you conform to 80 characters manually or is there a library that automatically wraps lines at 80?

Reply:
I'm using `set colorcolumn=79` in vim to highlight the PEP8 limit.